meilisearch-http/tests/common/server.rs

```diff
@@ -184,7 +184,7 @@ impl Server {
 pub fn default_settings(dir: impl AsRef<Path>) -> Opt {
     Opt {
         db_path: dir.as_ref().join("db"),
-        dumps_dir: dir.as_ref().join("dump"),
+        dump_dir: dir.as_ref().join("dump"),
```

Comment:
```suggestion
        dump_dir: dir.as_ref().join("dumps"),
```

I know this has nothing to do with your PR, but I just noticed a small inconsistency between the tests and the prod. The directory should be called `dumps` here :+1: